pulpcore/cli/common/openapi.py

```diff
@@ -14,7 +16,11 @@
 translation = get_translation(__name__)
 _ = translation.gettext
 
+UploadType = Union[bytes, IO[bytes]]
```

Comment:
Is there a reason not to get this from common/context?

Reply:
That would be a circular import.

And importing the other way around lead to "this was not explicitly exported" errors.

Reply:
OK, makes sense - can we get a comment that says "duplicated from common/context because of circular imports" so that Future Us will not forget and end up changing this in only one place.

Reply:
Maybe importing from two levels down is not _that_ bad. I changed the definitions a bit...